examples/sandbox/index.ts

```diff
@@ -921,7 +918,7 @@ $rippleAddr.on("click", async (e) => {
 $rippleTx.on("click", async (e) => {
   e.preventDefault();
   if (!wallet) {
-    $ethResults.val("No wallet?");
+    $rippleResults.val("No wallet?");
```

Comment:
This is the first of many copy/pasta errors with the `$ethResults` etc selector being used for the wrong coin